File: pangram/Java/PangramTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class PangramTest {
    private final Pangram pangram = new Pangram();

    @Test
    void checkPangram() {
        assertEquals("pangram", pangram.pangrams("We promptly judged antique ivory buckles for the next prize"));
    }

    @Test
    void checkNotPangram() {
        assertEquals("not pangram", pangram.pangrams("We promptly judged antique ivory buckles for the prize"));
    }

    @Test
    void checkEmpty() {
        assertEquals("not pangram", pangram.pangrams(""));
    }
}

File: pangram/JS/pangram.test.js
const pangram = require("./pangram");

test("check that is a pangram", () => {
  expect(pangram("We promptly judged antique ivory buckles for the next prize")).toBe("pangram");
})

test("check that is not a pangram", () => {
  expect(pangram("")).toBe("not pangram");
})

test("che that sentence is not a pangram", () => {
  expect(pangram("We promptly judged antique ivory buckles for the prize")).toBe("not pangram");
})

